crates/meroctl/src/cli/context/grant.rs

```diff
@@ -59,7 +60,7 @@ impl GrantPermissionCommand {
         make_request::<_, GrantPermissionResponse>(
             environment,
             &client,
-            multiaddr_to_url(multiaddr, "admin-api/dev/contexts/grant-permission")?,
+            multiaddr_to_url(multiaddr, "admin-api/dev/contexts/identity/grant-permission")?,
```

Comment:
this isn't the right url tho, is it?

<img src="https://github.com/user-attachments/assets/fbf5ca8d-4f53-49ac-a4f9-fa160835e316" width="50%">